2024/immersive-web-wg.html

```diff
@@ -252,8 +252,10 @@ <h3>
           Normative Specifications
         </h3>
         <p>
-          The Working Group will deliver the following W3C normative specifications and WebXR Living Standards as noted
-          under the module deliverables section:
+          The Working Group will deliver the following W3C normative specifications: <br>
+          Some specifications have their Expected W3C Recommendation as yearly basis, and 
+          Working Group will continue to maintain specifications to support new XR hardware as a living specification 
+          where Working Group will publish new Candidate Recommendations in timely after substantive changes. 
```

Comment:
```suggestion
          Some specifications are expected to be published as W3C Recommendations on a yearly basis, and 
          the Working Group will continue to maintain specifications to support new XR hardware as living specifications 
          where the Working Group will publish new Candidate Recommendations after substantive changes. 
```

Reply:
This is just a wording suggestion, which I hope matches the WG's intent. This paragraph doesn't need updating from my point of view, as the changes below to indicate the "Draft state" as CR all I was suggesting really.

Reply:
@chrisn Really thank you for correcting English. I believe we are happy with this.

@Yonet @AdaRoseCannon @cwilso  ok to go with this? If ok, I'll go to all responders.

Reply:
Approved